dockyard/common/container/container.py

```diff
@@ -0,0 +1,162 @@
+from dockyard.common import utils, link
+
+import json
+from pecan.core import redirect
+
+class Container(object):
+    def __init__(self):
+        pass
+
+    def list(self, name_id=None, query_params=None):
+        if name_id:
+            url = ('/containers/%s/json' % (name_id))
+        else:
+            url = '/containers/json'
+
+        if query_params:
+            url = ('%s?%s' % (url, query_params))
+
+        return utils.dispatch_get_request(url, 'http').data
+
+    def changes(self, name_id=None):
+        url = (('/containers/%s/changes') % (name_id))
```

Comment:
I think, we should make one more function, which will take variables defined somewhere(may be class variables) automatically and changes can be passed from here, it will reduce making urls directly as being done here. I think, that method can add more flexibility by having large number of options. It can be discussed.